base_substate/tests/__init__.py

```diff
@@ -0,0 +1,2 @@
+from . import models
```

Comment:
This will break your test, please read : https://github.com/akretion/odoo-test-helper